src/installer/__tests__/__snapshots__/getScript.ts.snap

```diff
@@ -28,12 +28,12 @@ if ! command -v node >/dev/null 2>&1; then
   echo \\"Can't find node in PATH, trying to find a node binary on your system\\"
 fi
 
-if [ -f $scriptPath ]; then
+if [ -f "$scriptPath" ]; then
   if [ -f ~/.huskyrc ]; then
     debug \\"source ~/.huskyrc\\"
     source ~/.huskyrc
   fi
-  node_modules/run-node/run-node $scriptPath $hookName \\"$gitParams\\"
+  node_modules/run-node/run-node "$scriptPath" $hookName \\"$gitParams\\"
```

Comment:
Just realized that it was failing because `"` isn't escaped, but adding `-u` to `npm test` should update them correctly :)